apps/mobile/src/components/Login/WalletSelectorBottomSheet.tsx

```diff
@@ -0,0 +1,142 @@
+
+import { BottomSheetRow } from '../Feed/Posts/PostBottomSheet';
+import {
+  GalleryBottomSheetModal,
+  GalleryBottomSheetModalType,
+} from '../GalleryBottomSheet/GalleryBottomSheetModal';
+import { useSafeAreaPadding } from '../SafeAreaViewWithPadding';
+import { Typography } from '../Typography';
+
+const SNAP_POINTS = ['CONTENT_HEIGHT'];
+
+type Props = {
+  onConnectWallet: () => void;
+  isSignedIn: boolean;
+};
+
+function WalletSelectorBottomSheet(
+  { onConnectWallet, isSignedIn }: Props,
+  ref: ForwardedRef<GalleryBottomSheetModalType>
+) {
+  const { bottom } = useSafeAreaPadding();
+
+  const bottomSheetRef = useRef<GalleryBottomSheetModalType | null>(null);
+
+  const { animatedHandleHeight, animatedSnapPoints, animatedContentHeight, handleContentLayout } =
+    useBottomSheetDynamicSnapPoints(SNAP_POINTS);
+
+  return (
+    <GalleryBottomSheetModal
+      ref={(value) => {
+        bottomSheetRef.current = value;
+
+        if (typeof ref === 'function') {
+          ref(value);
+        } else if (ref) {
+          ref.current = value;
+        }
+      }}
+      snapPoints={animatedSnapPoints}
+      handleHeight={animatedHandleHeight}
+      contentHeight={animatedContentHeight}
+    >
+      <View
+        onLayout={handleContentLayout}
+        style={{ paddingBottom: bottom }}
+        className="p-4 flex flex-col space-y-6"
+      >
+        <View className="flex flex-col space-y-4">
+          {isSignedIn ? (
+            <Typography
+              className="text-lg text-black-900 dark:text-offWhite text-center"
+              font={{ family: 'ABCDiatype', weight: 'Bold' }}
+            >
+              Please sign the message in your wallet
+            </Typography>
+          ) : (
+            <Typography
+              className="text-lg text-black-900 dark:text-offWhite"
+              font={{ family: 'ABCDiatype', weight: 'Bold' }}
+            >
+              Network
+            </Typography>
+          )}
+        </View>
+
+        <View className="flex flex-col space-y-2">
+          {isSignedIn ? (
+            <View
+              className="items-center"
+              style={{
+                paddingBottom: bottom,
+              }}
+            >
+              <SpinnerIcon spin />
+            </View>
+          ) : (
+            <BottomSheetRow
+              icon={
+                <View className="bg-white p-1 rounded-full">
+                  <EthIcon width={24} height={24} />
+                </View>
+              }
+              fontWeight="Bold"
+              text="Ethereum and L2s"
+              onPress={onConnectWallet}
+              rightIcon={<EvmStackedIcons />}
+            />
+          )}
+        </View>
+      </View>
+    </GalleryBottomSheetModal>
+  );
+}
+
+const ForwardedWalletSelectorBottomSheet = forwardRef(WalletSelectorBottomSheet);
+
+export { ForwardedWalletSelectorBottomSheet as WalletSelectorBottomSheet };
+
+function EvmStackedIcons() {
```

Comment:
small design comment, on dark mode we want the border around each icon to match the background - basically appear transparent

<img width="95" alt="CleanShot 2023-08-28 at 21 33 59@2x" src="https://github.com/gallery-so/gallery/assets/80802871/fd6b5135-4278-4235-90f1-189b9b370284">


design:
<img width="827" alt="CleanShot 2023-08-28 at 21 41 40@2x" src="https://github.com/gallery-so/gallery/assets/80802871/9a9edf92-6249-4db0-98b3-587fb95f1786">

Reply:
Updated on dark mode version!

<img width="417" alt="CleanShot 2023-08-29 at 08 46 28@2x" src="https://github.com/gallery-so/gallery/assets/4480258/dd9a0f5b-f401-458b-9184-27be12c4c134">